org.eu.wabarc.wayback.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I recommend moving appdata, desktop and icon files to your upstream repo and distribute them in the tarball this gets build from.